GliaWidgets/Sources/CallVisualizer/VideoCall/View/ConnectView/VideoCallView.ConnectView.swift

```diff
@@ -0,0 +1,149 @@
+import UIKit
+
+extension CallVisualizer.VideoCallView {
+    final class ConnectView: BaseView {
+
+        // MARK: - Properties
+
+        let operatorView = ConnectOperatorView()
```

Comment:
Please review access level of properties and methods in new files.

Reply:
fixed [00e2997](https://github.com/salemove/ios-sdk-widgets/pull/494/commits/00e29971b62e4284d2a0668fd9828bc9cb1e0b78)